centaur/src/main/resources/standardTestCases/sizerelativepath/1495bytes.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Broad Institute, Inc.
```

Comment:
I would suggest making this generic content rather than license-looking (eg could it just be1495 random characters?)

Reply:
`cat /dev/urandom | base32 | head --bytes 1495 > centaur/src/main/resources/standardTestCases/sizerelativepath/1495bytes.txt`

Done.